examples/oneapi/dpc/source/pca/pca_cor_dense_batch.cpp

```diff
@@ -20,29 +20,22 @@
 
```

Comment:
@vepifanov do you plan to update all examples with use of `data_source`?

Reply:
This is discussable - we can use data source in all examples or just introduce separate example specifically for data source functionality. I modified pca example for testing for now.